lib/framework/UploadFileService.cpp

```diff
@@ -122,18 +124,17 @@ void UploadFileService::uploadComplete(AsyncWebServerRequest * request) {
         request->send(response);
         return;
     }
+
     if (strlen(md5) == 32) {
         auto *     response = new AsyncJsonResponse(false, 256);
         JsonObject root     = response->getRoot();
         root["md5"]         = md5;
         response->setLength();
         request->send(response);
-        // AsyncWebServerResponse * response = request->beginResponse(201, "text/plain", md5); // created
-        // request->send(response);
         return;
     }
 
-    handleError(request, 403); // send the forbidden response
+    handleError(request, 406); // send the forbidden response
```

Comment:
You should also change the comment, forbidden is 403, 406 is not acceptable

Reply:
yes, will change